### PR TITLE
fix symmetry bug

### LINKED ISSUE #180: z-axis rotational symmetry
Wouter mentioned the rotational symmetry might be incorrect. Currently it is applied along the Z1-axis, but his point was it should be applied along the Z2-axis as the ZXZ angles specify the rotation of the reference onto the particle in the tomogram. Thinking about it he is probably right.

Perhaps fix or remove argument, depending on time?

@sroet maybe you could look at it? Should I assign it?

### PR DESCRIPTION
**Pull request by sroet**
closes #180

### CODE REVIEW

File: src/pytom_tm/angles.py [hunk 1]
```diff
@@ -88,6 +88,8 @@ def get_angle_list(angle: Union[pathlib.Path, float], sort_angles: bool = True,
           maximum difference (in degrees) for the angle list
     sort_angles: bool, default True
         sort the list, using python default angle_list.sort(), sorts first on Z1, then X, then Z2
+    symmetry: int, default 1
+        the returned list will only have Z2 angles [0, (2*pi/symmetry))
```
Comment (McHaillet) [inline src/pytom_tm/angles.py:92]: Could you add here that the symmetry is not applied for files? A similar message would also be handy to have at the command line argument for rotational symmetry.

**Review by McHaillet — CHANGES_REQUESTED** [review]
LGTM, just one comment about docs and a suggestion for some user info.

File: src/pytom_tm/angles.py (hunk 1 above)
Comment (sroet) [reply src/pytom_tm/angles.py:92]: But it is applied to files?
files are read in in line `118` and then filtered in line `122`?
(this was also the case in the old implementation, hence why I left it that way

Comment (McHaillet) [reply src/pytom_tm/angles.py:92]: Woops, read it wrong!

**Review by McHaillet — APPROVED** [review]
Good to go then

**Pull request merged**